examples/NewRelic.LogEnrichers.NLog.Examples/Program.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using NewRelic.Api.Agent;
+using NLog;
+using NLog.Config;
+using NLog.Targets;
+
+
+namespace NewRelic.LogEnrichers.NLog.Examples
+{
+	class Program
+	{
+        private static Logger _logger;
+
+        static void Main(string[] args)
+		{
+            Console.WriteLine("Welcome to the New Relic Logging Extentions for NLog");
+            Console.WriteLine();
+
+
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Usage:");
+                Console.WriteLine("dotnet run {OutputFolderPath}");
+                return;
+            }
+
+            var folderPath = string.Join(' ', args);
+
+            var folderPath_StandardLogs = Path.Combine(folderPath, "StandardLogs");
+            var folderPath_NewRelicLogs = Path.Combine(folderPath, "NewRelicLogs");
+
+            // Ensure that the output folders exist
+            Directory.CreateDirectory(folderPath_StandardLogs);
+            Directory.CreateDirectory(folderPath_NewRelicLogs);
+
+            // Create Logger
+            _logger = CreateLogger(
+                folderPath_StandardLogs: folderPath_StandardLogs,
+                folderPath_NewRelicLogs: folderPath_NewRelicLogs);
+
+            // This log information will be visible in New Relic Logging. Since 
+            // a transaction has not been started, this log message will not be
+            // associated to a specific transaction.
+            _logger.Info("Hello, welcome to the Nlog Logs In Context sample app!");
+
+            do
+            {
+                Console.WriteLine("Creating Logged Transactions");
+
+                // Call three example methods that create transactions
+                TestMethod("First Transaction");
+                TestMethod("Second Transaction");
+                TestMethod("Third Transaction");
+
+                Console.WriteLine("Press <ENTER> to continue, Q to exit.");
+            }
+            while (Console.ReadLine() != "Q");
+
+            // This log information will be visible in New Relic Logging. Since 
+            // a transaction has not been started, this log message will not be
+            // associated to a specific transaction.
+            _logger.Info("Thanks for visitng, please come back soon!");
```

Comment:
```suggestion
            _logger.Info("Thanks for visiting, please come back soon!");
```

Reply:
Since this typo was copy/pasted from the Serilog example, I did a commit that fixes it in both places.